test/tests/functional/pbs_suspend_resume_accounting.py

```diff
@@ -178,27 +178,29 @@ def test_suspend_resume_job_scheduler(self):
         record = 'r;%s;' % jid1
         self.server.accounting_match(msg=record, id=jid1)
 
+    @skipOnCpuSet
     def test_admin_suspend_resume_signal(self):
         """
         Test case to verify accounting of admin-suspend
         and admin-resume records.
         """
+        a = {ATTR_rescavail + '.ncpus': 2}
+        self.server.create_vnodes('vn', a, 1, self.mom)
```

Comment:
can we avoid creating vnodes on cpuset machined, so that we can remove the skipOnCpuSet?